backend/common/config.py

```
@@ -18,9 +18,16 @@ class Settings(BaseSettings):
         extra="ignore"
     )
 
+    # App Config
+    environment: str = "development"
     database_url: str = f"sqlite:///{BASE_DIR / 'app.db'}"
     secret_key: str = "super-secret-key"
```

Ruff (0.14.6)

Possible hardcoded password assigned to: "secret_key"

(S105)